aospy/data_loader.py

```diff
@@ -10,6 +10,26 @@
 from .utils import times, io
 
 
+def preprocess_and_rename_grid_attrs(func):
+    """Call a custom preprocessing method first then
+    ``grid_attrs_to_aospy_names``
+    
+    Parameters
+    ----------
+    func : function
+       An arbitrary function to call before calling
+       ``grid_attrs_to_aospy_names`` in ``_load_data_from_disk``.  Must take
+       only an xr.Dataset as an argument.
+    
+    Returns
+    -------
+    function
```

Comment:
Describe.  Can be essentially same language as in extra sentence I ask for above.